README.md

```diff
@@ -96,6 +96,8 @@ Edit the default configuration of the server by adding options to your **laravel
 | `databaseConfig` |  `{}`    |  Configurations for the different database drivers [Example](#database)|
 | `hostname`     | `http://localhost` | The host of the socket.io server ex.`app.dev` |
 | `port`         | `6001`         | The port that the socket.io server should run on |
+| `protocol`     | `http`         | either `http` or `https` |
+| `referrers`     | `[]`         | ex.`['abc.com','xyz.com']` check [Referrers](#referrers) |
```

Comment:
okay, can u plz fix it from ur end because i never used it , i just added it like that from what i understood from the referrer section.